vllm/config/parallel.py

```diff
@@ -141,6 +144,10 @@ class is dynamically inherited by the worker class. This is used to inject
     """ Use data parallelism instead of tensor parallelism for vision encoder.
     Only support LLama4 for now"""
 
+    xfer_handshake_metadata: Optional[dict[int, dict[
+        int, "KVConnectorHandshakeMetadata"]]] = None
```

Comment:
It's a tiny change, but since you have put `KVConnectorHandshakeMetadata = Any` in the else block above, it's safe to remove the `"`:

```suggestion
        int, KVConnectorHandshakeMetadata]]] = None
```